app/models/file_depot.rb

```diff
@@ -33,7 +33,7 @@ def upload_file(file)
     @file = file
   end
 
-  def merged_uri
+  def merged_uri(uri, _api_port)
```

Comment:
Doesn't this change the behavior? Before it was returning the attribute, now it's returning the parameter. Is that what we want?

Reply:
It should be the same, but I realize there may be cases where its not - so we can ignore the parameter (via ```_uri```).